theory.MD

```diff
@@ -0,0 +1,34 @@
+**Question 1**
+*Fill in the blank:*
+Git is an example of a version control system.
```

Comment:
**Theory marks** 
💕Q1 - 2/2 (pull requst and file readability)

💕Q2 - 11/12, nearly perfect, don't forget about the `-m` flag for a `git commit`